vars/deployCluster.groovy

```diff
@@ -82,7 +82,16 @@ def call() {
                 }
             }
             else {
-                 env.BASTION_IP=sh(returnStdout: true, script: "cd ${WORKSPACE}/deploy && make terraform:output TERRAFORM_DIR=.${TARGET} TERRAFORM_OUTPUT_VAR=bastion_ip | grep -Eo '[0-9]{1,3}(\\.[0-9]{1,3}){3}'").trim()
+                sh '''#!/bin/bash
+                    cd ${WORKSPACE}/deploy
+                    OPENSHIFT_POWERVC_DEPLOY_DIR=.deploy-openshift4-powervc/
+                    TERRAFORM_VARS_FILE_POWERVC=.deploy-openshift4-powervc.tfvars
+                    grep '^bootstrap*' $TERRAFORM_VARS_FILE_POWERVC | sed 's|"count"   = 1|"count"   = 0|g' $TERRAFORM_VARS_FILE_POWERVC > temp_vars.tfvars
+                    mv -f temp_vars.tfvars $OPENSHIFT_POWERVC_DEPLOY_DIR/$TERRAFORM_VARS_FILE_POWERVC
```

Comment:
can't `sed -i` be directly used here ?

Reply:
@sudeeshjohn Removed the grep and only used sed